CefSharp.Example/CefExample.cs

```diff
@@ -70,11 +70,11 @@ public static void Init()
             
             settings.LogSeverity = LogSeverity.Verbose;
 
-            if (DebuggingSubProcess)
-            {
-                var architecture = Environment.Is64BitProcess ? "x64" : "x86";
-                settings.BrowserSubprocessPath = "..\\..\\..\\..\\CefSharp.BrowserSubprocess\\bin\\" + architecture + "\\Debug\\CefSharp.BrowserSubprocess.exe";
-            }
+            //if (DebuggingSubProcess)
```

Comment:
Possible to revert these changes? I can take care of it after merge if it's a problem